app/views/catalog/_bookmark_all.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="checkbox bookmark_all">
+<div class="checkbox bookmark_all" tabindex='0'>
```

Comment:
Could we use a button, rather than a div here?  Then we could listen for a click event to catch either the space bar or the enter key.  It would also get added to the tab order automatically, without having to specify tabindex.

Reply:
Thanks for looking into this!  I think the button would be better long term (we wouldn't need to remember the tabindex if we refactor this code in the future, for example), if you can get the CSS sorted.  Another benefit is that the button would be navigable by assistive technologies that allow you to navigate by form control (like NVDA).